src/chunks/find-jobs-template-chunk/find-all-jobs-template-chunk/FindAllJobsTemplateChunk.tsx

```diff
@@ -128,7 +128,7 @@ const FindAllJobsTemplateChunk = () => {
 
         <div className="flex my-1">
           <Button
-            overrideStyles="w-[100%] h-[50px] bg-[#0046BF] flex lg:hidden text-center items-center rounded-none
+
```

Comment:
hope this doesnt affect the current design?

Reply:
No it won't